packages/frontend/src/components/accounts/ledger/SignInLedgerViews/Authorize.js

```diff
@@ -22,14 +18,9 @@ const Authorize = ({
             <h1><Translate id='signInLedger.header' /></h1>
             <Translate id='signInLedger.one' />
             <br /><br />
-            <LocalAlertBox localAlert={status.localAlert} />
             <LedgerHdPaths
-                path={path}
-                onSetPath={(path) => setPath(path)}
-                onConfirmHdPath={() => {
-                    setConfirmedPath(path);
-                    Mixpanel.track('IE-Ledger Sign in set custom HD path');
```

Comment:
We're no longer firing this event, is this no longer needed?

Reply:
In the previous implementation, we were triggering it on every path change (not only approval), so it was pretty useful. I added it to `setConfirmedPath` now so it will be triggered when the path is actually set.